source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -17,31 +17,39 @@ The global vulnerability database is created automatically, currently pulling da
-This improves the performance and avoids repeated alerts during a configurable time. We have then three different types of scan:
+It can run a scan on startup (:ref:`run_on_start <vuln_det_run_on_start>`) and every certain period of time (:ref:`interval <vuln_det_interval>`).
+In any of these cases, the packages and the operating system that were already scanned will not be re-scanned, only the new ones will do and alerts are going to be generated.
+If the database of vulnerabilities receives new CVEs information and :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires all packages, as well as the operating system, will be re-scanned.
 
-- Baseline: This scan type will be triggered the first time after Vulnerability Detector is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
-- Full scan: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the newly detected vulnerabilities and the ones that no longer affect the agent will be alerted.
-- Partial scans: Only new packages are scanned and the vulnerabilities detected are alerted.
+We have then three different types of scan:
+
+- `Baseline`: This scan type will be triggered the first time after `Vulnerability Detector` is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
+- `Full scan`: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the vulnerabilities update will be alerted.
+- `Partial scan`: Only new packages are scanned and, as it is in `Full scans`, any vulnerability update will be reported.
 
 There are few considerations that arise from this behavior:
 
-- The user can not trigger a full scan manually. The only option is to disable and enable Vulnerability Detector to trigger the baseline scan. For that purpose, a service restart is needed.
-- The partial scans generate alerts for new packages, but they do not delete alerts for removed packages.
-- Partial scans can be triggered with a Manager restart.
+- The `partial scan` can be triggered with a manager restart and generate alerts for new packages, as well as alerts for removed packages.
```

Comment:
Not only partial scans can be triggered after a manager restart. 
i.e. The first restart will trigger a baseline.
And a full_scan can be triggered if the conditions match.

Reply:
Good point